mitosheet/mitosheet/step_performers/user_defined_import.py

```diff
@@ -0,0 +1,91 @@
+
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+from time import perf_counter
+from typing import Any, Dict, List, Optional, Set, Tuple
+
+import pandas as pd
+
+from mitosheet.code_chunks.code_chunk import CodeChunk
+from mitosheet.code_chunks.user_defined_import_code_chunk import \
+    UserDefinedImportCodeChunk
+from mitosheet.state import DATAFRAME_SOURCE_IMPORTED, State
+from mitosheet.step_performers.step_performer import StepPerformer
+from mitosheet.step_performers.utils import get_param
+from mitosheet.types import ColumnID
+
+
+class UserDefinedImportStepPerformer(StepPerformer):
+    """
+    Allows a user to use one of the user_defined_importers.
+    """
+
+    @classmethod
+    def step_version(cls) -> int:
+        return 1
+
+    @classmethod
+    def step_type(cls) -> str:
+        return 'user_defined_import'
+
+    @classmethod
+    def execute(cls, prev_state: State, params: Dict[str, Any]) -> Tuple[State, Optional[Dict[str, Any]]]:
+        print("PARAMS", params)
+        importer: str = get_param(params, 'importer')
```

Comment:
its unclear from this variable name what `importer` is. Is it the name of the function? Is it the actual import code?